drivers/led_strip/Kconfig.ws2812

```diff
@@ -30,20 +30,49 @@ config WS2812_STRIP_I2S
 
 config WS2812_STRIP_GPIO
 	bool "WS2812 LED strip GPIO driver"
-	# Only an Cortex-M0 inline assembly implementation for the nRF51
-	# is supported currently.
+	# Only an Cortex-M inline assembly implementation for the nRF91, nRF51,
+	# nRF52 and nRF53 is supported currently.
 	default y
 	depends on DT_HAS_WORLDSEMI_WS2812_GPIO_ENABLED
-	depends on SOC_SERIES_NRF51X
+	depends on (SOC_SERIES_NRF91X || SOC_SERIES_NRF51X || SOC_SERIES_NRF52X || SOC_SERIES_NRF53X)
 	select LED_STRIP_RGB_SCRATCH
 	help
-	  Enable driver for WS2812 (and compatibles) LED strip directly controlling with GPIO.
-	  The GPIO driver does bit-banging with inline assembly,
-	  and is not available on all SoCs.
+	  Enable driver for WS2812 (and compatibles) LED strip directly
+	  controlling with GPIO. The GPIO driver does bit-banging with inline
+	  assembly, and is not available on all SoCs.
 
 	  Note that this driver is not compatible with the Everlight B1414
 	  controller.
 
+if WS2812_STRIP_GPIO
+
+config DELAY_T1H
+	int
```

Comment:
Note that the prompt is missing on all this options. And then the `bbc_microbit` board configuration of the `led_strip` sample can't be used:
```
$ west build -b bbc_microbit samples/drivers/led_strip
...
Merged configuration '/home/simon/src/zephyrproject/zephyr/samples/drivers/led_strip/boards/bbc_microbit.conf'

error: DELAY_T1H (defined at drivers/led_strip/Kconfig.ws2812:49) is assigned in a configuration
file, but is not directly user-configurable (has no prompt). It gets its value indirectly from other
symbols. See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_DELAY_T1H and/or look up
DELAY_T1H in the menuconfig/guiconfig interface. The Application Development Primer, Setting
Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful
too.

CMake Error at /home/simon/src/zephyrproject/zephyr/cmake/modules/kconfig.cmake:389 (message):
  command failed with return code: 1
...
```

Reply:
I'll make sure this fix this, and do some better testing next time ;-)